tests/lib/rules/strict-component-boundaries.js

```diff
@@ -57,15 +57,5 @@ ruleTester.run('strict-component-boundaries', rule, {
       parserOptions,
       errors,
     },
-    {
-      code: `import someThing from '../SomeComponent/fixtures/SomeMockQuery/query.json';`,
```

Comment:
So this import would now be valid? We certainly don't want it but maybe it just never comes up?

Reply:
It would be and yeah I just don't think it comes up. I may come back to this.